Docs/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 
 ### Safety
 
+### Documentation
+- Added Getting Started example to use in public documentation to show basics configuration steps
```

Comment:
What do you think about the notion of "Code Snippets"? They are not a part of this app, just added here for compilation and API check.
```suggestion
- Added Getting Started code snippet showing basic SDK configuration steps
```